scheduler/src/cook/config.clj

```diff
@@ -444,12 +444,23 @@
                                   :default-pool "no-pool"}
                                  pool-selection)})))
      :kubernetes (fnk [[:config {kubernetes {}}]]
-                   (merge {:default-workdir "/mnt/sandbox"
-                           :pod-condition-containers-not-initialized-seconds 120
-                           :pod-condition-unschedulable-seconds 60
-                           :reconnect-delay-ms 60000
-                           :set-container-cpu-limit? true}
-                          kubernetes))}))
+                   (let [{:keys [controller-lock-num-shards]
+                          :or {controller-lock-num-shards 32}}
+                         kubernetes
+                         lock-objects
+                         (repeatedly
+                           controller-lock-num-shards
+                           #(Object.))]
+                     (log/info "Doing" controller-lock-num-shards "way lock-sharding in k8s")
```

Comment:
Why are we logging this and not other parameters? Isn't it all set in :settings anyways. We can also copy the value over, explicitly, so its in :settings instead of requiring the user to count the number of Object's in the array?

Reply:
Good points, will do

Reply:
Done